lib/dqt/api/v1.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative "v1/dqt_record"
+
+module Dqt
+  class Api
+    class V1
+      def initialize(client:)
+        self.client = client
+      end
+
+      def dqt_record
+        @dqt_record ||= DQTRecord.new(client: client)
+      end
+
+    private
+
+      attr_accessor :client
+    end
+  end
```

Comment:
What is the purpose of this? So far as I can see, the purpose is caching the response of `dqt_record`, cause we pass `client` through initialization firther down to DQTRecord. Is that something we need a separate class for?